scripts/storagedrawers.zs

```diff
@@ -11,12 +11,12 @@ mods.jei.JEI.removeAndHide(<framedcompactdrawers:framed_compact_drawer>);
 
 var drawerWoodTypeMap as IItemStack[][] =
 [
-[<minecraft:planks:0>, <storagedrawers:trim:0>, <quark:custom_chest:0>],
-[<minecraft:planks:1>, <storagedrawers:trim:1>, <quark:custom_chest:1>],
-[<minecraft:planks:2>, <storagedrawers:trim:2>, <quark:custom_chest:2>],
-[<minecraft:planks:3>, <storagedrawers:trim:3>, <quark:custom_chest:3>],
-[<minecraft:planks:4>, <storagedrawers:trim:4>, <quark:custom_chest:4>],
-[<minecraft:planks:5>, <storagedrawers:trim:5>, <quark:custom_chest:5>]
+[<minecraft:planks:0>, <storagedrawers:trim:0>],
+[<minecraft:planks:1>, <storagedrawers:trim:1>],
+[<minecraft:planks:2>, <storagedrawers:trim:2>],
+[<minecraft:planks:3>, <storagedrawers:trim:3>],
+[<minecraft:planks:4>, <storagedrawers:trim:4>],
+[<minecraft:planks:5>, <storagedrawers:trim:5>]
 ];
```

Comment:
Instead of doing something like this, we can instead change it to: `val planks = <minecraft:planks>.definition as IItemDefinition;`

Then, for any time you need a recipe that works will all planks, you can still use a for loop like this:
```zenscript
for i in 0 to 6 {
    planks.makeStack(i)
}
```
This will allow you to clean things up significantly. `makeStack()` makes an IItemStack out an IItemDefinition with a metadata value. You may need additional imports, and my syntax might not be perfectly correct, so be sure to validate these things before committing etc.